apps/builder/env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="vite/client" />
```

Comment:
why just not add to tsconfig types?

Reply:
I honestly don't know. This is what they recommend to do

Reply:
will fix in another PR